### ISSUE TITLE
User Interface Concept - Create Universal Component

### ISSUE THREAD

**Issue by REllEK-IO** [Phuire-Research/logixUX#60]
Via the MVP logixUX. Informed the need for "Universal Components," where these components could exist between different Pages. While sharing the same exact component. The issue with the current implementation is that a "Sidebar," component is being replicated on each page.

So to improve this process. I would need to add a component property to the User Interface Concept. That a page could then reference from State, to be added into its own composition when required.

Another example would be implementing our own Control Bar within the application. As that too would be replicated upon each page load.

The main issue is triggering multiple renders and atomic edits. After this change, if a component finds itself issuing multiple atomic edits. That component would need to be promoted to a "Universal Component."

**Comment by REllEK-IO** [comment]
This will take priority over exports for now.

**Comment by REllEK-IO** [comment]
Added a simple flag to enable the universal component functionality.
Working on this angle is revealing where branch prediction is working us over. Specifically in regards to time complexity.
As in the current client onChange principle plan. I should be able to use the updateUnifiedKeyedSelector helper function to ensure that we are working within the desired unified scope. But instead I have to hard code in that we are working in 'userInterfaceClient.'

Both accomplish the same objective. But one has a high failure rate in regards to that branch being fully integrated into the application. So strange to think that probabilities arise in such a simple program at this current scale. 

The next goal to work around this issue is to finally multi thread this process and embrace the P/NP out of bounds that this application is able to encompass.

Time Complexity + Branch Prediction. At the very least I know have a guideline to set up for users plans. That anything that might go into that cubed time should be reduced to squared. But that likewise excludes that I have ran into the same issue even when adding in an additional loop on the base N layer.

Really does bring in the importance of establishing specific symbols that are provably terminating in this system. As at least there I can guarantee chunk what is capable of halting within this faulty system of computation. And know now why so many years ago I looked at my first Unified Turing machine in horror. Could just be stretching NodeJS to its limits. But likewise it shouldn't be running into issues with a simple loop being added in to change a single parameter on an object that would not effect a race condition.
